src/main/java/edu/ucla/library/prl/harvester/services/HarvestJobSchedulerServiceImpl.java

```diff
@@ -37,7 +118,90 @@ public Future<Void> removeJob(final int aJobId) {
 
     @Override
     public Future<Void> close() {
-        // TODO implement method
-        return Future.succeededFuture(null);
+        try {
+            myScheduler.shutdown();
+
+            return Future.succeededFuture();
+        } catch (final SchedulerException details) {
+            return Future.failedFuture(details);
+        }
+    }
+
+    /**
+     * Initializes the scheduler with all of the {@link Job}s stored in the database.
+     *
+     * @return A Future that succeeds if the saved jobs were restored
+     */
+    @SuppressWarnings("rawtypes")
+    private Future<Void> initializeScheduler() {
+        return myHarvestScheduleStoreService.listJobs().compose(jobs -> {
+            return CompositeFuture.all(jobs.stream().map(job -> (Future) scheduleJob(job, false)).toList());
+        }).mapEmpty();
+    }
+
+    /**
+     * @param aJob A job
+     * @param aReplaceIfExists Whether or not to overwrite the job if it already exists
+     * @return A Future that succeeds if the job was added to, or updated in, the scheduler
+     */
+    private Future<Void> scheduleJob(final Job aJob, final boolean aReplaceIfExists) {
+        final JobKey key = new JobKey(Integer.toString(aJob.getID().get()));
+        final JobDetail jobDetail = JobBuilder.newJob(RunHarvest.class).withIdentity(key)
+                .usingJobData(ENCODED_JOB_JSON, aJob.toJson().encode()).build();
+        final CronScheduleBuilder scheduleBuilder = CronScheduleBuilder.cronSchedule(aJob.getScheduleCronExpression());
+        final CronTrigger trigger =
+                TriggerBuilder.newTrigger().withSchedule(scheduleBuilder).startNow().endAt(null).build();
+
+        try {
+            myScheduler.scheduleJob(jobDetail, Set.of(trigger), aReplaceIfExists);
+
+            return Future.succeededFuture();
+        } catch (final SchedulerException details) {
+            return Future.failedFuture(details);
+        }
+    }
+
+    /**
+     * Runs a harvest job, then publishes its result (or resulting error) to the event bus addresses
+     * {@link JOB_RESULT_ADDRESS} and {@link ERROR_ADDRESS}, respectively.
+     */
+    public static final class RunHarvest implements org.quartz.Job {
+
+        /**
+         * Per the docs for {@link org.quartz.Job}, a public no-argument constructor is required.
```

Comment:
Interesting and odd that the default one wouldn't work. Thanks for the note here.